### PR TITLE
Add support for Preact X

### PR DESCRIPTION
**Pull request by filoozom**
I'm not exactly sure if that's enough, but it seems to be working for now.

Eh, the tests seem to disagree 😛

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,6 +91,6 @@
     "dlv": "^1.1.2"
   },
   "peerDependencies": {
-    "preact": "*"
+    "preact": ">=10 || ^10.0.0-alpha.0"
```
Comment (billneff79) [inline package.json]: We still want to support as much of the preact version history as possible, so this _should_ either continue to be `*` if we can support every version, or at a minimum something like `>=6` or some minimally tested version that we know it works with.

- [ ] Fix peer dependency version range

**Comment by billneff79** [comment]
It looks like you missed the `children` used in the `localizer.js` file.

**Comment by billneff79** [comment]
It's possible that something might break in `translate-mapping.js`as well with it's use of children of Text nodes on line 30.

**Comment by filoozom** [comment]
Closing in favour of #33

**Pull request closed without merging**